sync/validation_queue.go

```diff
@@ -163,6 +164,9 @@ func (vq *blockQueue) updateDependencies(block types.Hash32, valid bool) {
 // dependencies can be of type block/layer
 // for block jobs we need to return  a list of finished blocks
 func (vq *blockQueue) removefromDepMaps(block types.Hash32, valid bool, doneBlocks []types.Hash32) []types.Hash32 {
+	vq.invalidate(block, valid)
```

Comment:
I'd write `vq.fetchQueue.invalidate` since this is confusing

Reply:
good point